index.js

```diff
@@ -67,10 +64,10 @@ S3Storage.prototype._handleFile = function (req, file, cb) {
         if (ev.total) currentSize = ev.total
       })
 
-      upload.send(function (err) {
+      upload.send(function (err, data) {
         if (err) return cb(err)
 
-        cb(null, { size: currentSize, key: filePath })
+        cb(null, extend(data, { size: currentSize, key: key }))
```

Comment:
Style guide aside, being that this is an interface with Amazon's S3 service the capitalization is how Amazon's API is constructed I was trying to keep it consistent despite its inconsistency. However, I don't have any strong feelings about its capitalization. I can change that and `data` parameter name.